src/main/java/org/broadinstitute/hellbender/engine/dataflow/SAMSerializableFunction.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.SerializableFunction;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SAMTextHeaderCodec;
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.util.StringLineReader;
+
+import java.io.Serializable;
+import java.util.function.Function;
+
+
+public class SAMSerializableFunction<O> implements SerializableFunction<Read, O> {
```

Comment:
is this class intended for subclassing? if yes, document it otherwise make final